### ISSUE TITLE
experimental results

### ISSUE THREAD

**Issue by PushineLee** [MayYoY/PhysFormer#1]
I am confused whether the experimental results of PhysFormer are reproducible. Have you successfully reproduced the same experimental results?

**Comment by MayYoY** [comment]
I can't fully reproduce that and I just tried my best to code this repository :(

**Comment by PushineLee** [comment]
I still can't get the bvp label in the train.txt file provided by the author in github. The training results of bvp labels obtained by my own interpolation are poor.

**Comment by MayYoY** [comment]
I have noticed this difference. It seems that the author used a bandpass filter to preprocess the bvp signals, but I can't improve the results in this way.

**Comment by mctosima** [comment]
Hi. I'm currently also replicating physformer experiments with my datasets. I can train the model by doing transfer learning using the pre-trained model. And I got 4 MAE with my datasets. Pretty good.

However,  something strange is, I can't do training from scratch. Even with hundreds of epochs, it is still not convergent. IDK if I miss something. Let's say I miss something, but why does my transfer learning work? In transfer learning, I could get from 30 MAE in 1st epoch to 4 MAE in 25 epochs.

My own dataset consist more data than VIPL-HR

**Training from scratch**
<img width="469" alt="image" src="https://user-images.githubusercontent.com/22334778/231989957-70295cd8-1035-4a48-b666-b8358fb2a9c8.png">

**Transfer Learning**
<img width="470" alt="image" src="https://user-images.githubusercontent.com/22334778/231996734-0850024e-cbb6-4a4a-9d20-10e0023a0a42.png">

**Comment by MayYoY** [comment]
> Hi. I'm currently also replicating physformer experiments with my datasets. I can train the model by doing transfer learning using the pre-trained model. And I got 4 MAE with my datasets. Pretty good.
> 
> However, something strange is, I can't do training from scratch. Even with hundreds of epochs, it is still not convergent. IDK if I miss something. Let's say I miss something, but why does my transfer learning work? In transfer learning, I could get from 30 MAE in 1st epoch to 4 MAE in 25 epochs.
> 
> My own dataset consist more data than VIPL-HR
> 
> **Training from scratch** <img alt="image" width="469" src="https://user-images.githubusercontent.com/22334778/231989957-70295cd8-1035-4a48-b666-b8358fb2a9c8.png">
> 
> **Transfer Learning** <img alt="image" width="470" src="https://user-images.githubusercontent.com/22334778/231996734-0850024e-cbb6-4a4a-9d20-10e0023a0a42.png">

Sorry, I haven't tried such a task. But I think it might be a problem with your dataset
